hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/cluster/ClusteringPlanPartitionFilter.java

```diff
@@ -42,13 +42,13 @@
  */
 public class ClusteringPlanPartitionFilter {
 
-  public static List<String> filter(List<String> partitions, HoodieWriteConfig config) {
+  public static List<String> filter(List<String> partitions, HoodieWriteConfig config, ArrayList<String> missingPartitions) {
```

Comment:
Maybe we return a Pair for both missing partitions and filtered partitions.

Reply:
make sense.
changed.